### PR TITLE
LPS-120411 - Add color to btn-outline-secondary token

### PR DESCRIPTION
**Pull request by marcoscv-work**
This PR adds a new token `text color` for `btn-outline-secondary`

--- 

How to test it:

- Product menu
- Open style
- Create a new stylebook
- Go to the button tab and see the new "color"

image

### CODE REVIEW

**Comment by jbalsas** [comment]
Hey @marcoscv-work, once you're done "editing", maybe you can squash these 2 commits in one? Doesn't make sense to have a SF commit right after the code you've just changed 😉

**Comment by marcoscv-work** [comment]
> Hey @marcoscv-work, once you're done "editing", maybe you can squash these 2 commits in one? Doesn't make sense to have a SF commit right after the code you've just changed 😉

But is other different variables!

**Comment by jbalsas** [comment]
> But is other different variables!

Yeah. It was the fact that you added your `btn` classes below the `portlet` variables just to then move the `portlet` variables below something else... so you could've placed the `btn` classes above and leave it at that if you wanted.

**Comment by marcoscv-work** [comment]
done!

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
> done!

It was probably better the other way 😂

**Comment by marcoscv-work** [comment]
> > done!
> 
> It was probably better the other way 😂

Do you want me to stop the test and update to two commits?

**Comment by jbalsas** [comment]
> > > done!
> > 
> > 
> > It was probably better the other way 😂
> 
> Do you want me to stop the test and update to two commits?

No please, no

**Review by eduardoallegrini — APPROVED** [review]

**Comment by jbalsas** [comment]
ci:test:relevant

File: modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json [hunk 1]
```diff
@@ -521,6 +521,19 @@
 							"name": "btnOutlineSecondaryBorderColor",
 							"type": "String"
 						},
+						{
+							"defaultValue": "#272833",
```
Comment (pat270) [inline modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json:525]: Just curious if this is the right color for `btn-outline-secondary`, I think I might have broke it during the Bootstrap 4 take over. It used to be `#6B6C7E` before that pr I think.

Comment (marcoscv-work) [reply modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json:525]: Wow, we have a lot of `#272833` in the `frontend-token-definition.json`.
We can adjust this in other PR

**Comment by marcoscv-work** [comment]
ci:forward

**Pull request closed without merging**